### DISCUSSION TITLE
Ошибка при экспорте анимации в OMF.

### DISCUSSION THREAD

**Discussion by SaberShoker** [PavelBlend/blender-xray#429] [Q&A] [answered]
При экспорте в OMF, появляется следующая ошибка.
![image](https://user-images.githubusercontent.com/69687233/135596071-0d553598-b13e-4d15-a112-54354f6eb29f.png)

**Comment by PavelBlend** [comment]
@SaberShoker нужно, чтобы скелет не имел пустых bone groups или лишних bone groups, которые не содержат неэкспортируемые кости. Я исправлю позже эту ошибку. А пока удали лишние bone groups у скелета в окне properties:
![01](https://user-images.githubusercontent.com/7983249/135623433-3293255f-e142-411c-8de6-fe649b4fe8bc.jpg)

**Comment by PavelBlend** [comment, marked as answer]
@SaberShoker ошибку исправил. Но пока не делал релиз, так как другую вещь не доделал. Можешь скачать незарелизеную версию аддона из последнего коммита:
https://github.com/PavelBlend/blender-xray/tree/733949c6bf96d949803f2a646b32eb9c68221814

**Comment by SaberShoker** [comment]
Большое спасибо
